test/lib/formatter/RuleResultRecombinator.test.ts

```diff
@@ -1291,9 +1291,9 @@ describe('RuleResultRecombinator', () => {
 		});
 
 		describe('Output Format: HTML', () => {
-			it ('Using --verbose-violations', async () => {				
+			it ('Using --verbose-violations', async () => {
 				const results = (await RuleResultRecombinator.recombineAndReformatResults(retireJsVerboseViolations, OUTPUT_FORMAT.HTML, new Set(['retire-js']), true)).results as string;
-				const violationString = results.split("const violations = [")[1].split("];\n")[0];
+				const violationString = results.split("const violations = [")[1].split("];")[0];
```

Comment:
This was failing in Windows, on Powershell. I believe the problem was that because Windows uses `\r\n` instead of `\n`, the split was failing.